internal/controller/securemgr/verifier/verifier.go

```diff
@@ -152,7 +153,7 @@ func addHashToContainerWhiteList(hash string) error {
 	fileContent, err := ioutil.ReadFile(cwlFilePath)
 	if err != nil {
 		fileContentStr := hash + "\n"
-		err = ioutil.WriteFile(cwlFilePath, []byte(fileContentStr), 0666)
+		err = ioutil.WriteFile(cwlFilePath, []byte(fileContentStr), 0666) // lgtm [go/path-injection]
```

Comment:
I think this is not CWE-117 and the `hash` should be verified in the previous [function](https://github.com/lf-edge/edge-home-orchestration-go/blob/c5e39bd14ab104f9f9b84fd0a1b7c1874e69ceca/internal/restinterface/externalhandler/externalhandler.go#L337) in advance.
What do you think?

Reply:
@t25kim Thank you for your opinion. This remark does not apply to the CWE-117 (as I mentioned in the text of the PR). **I think your idea is great.** I have already made a draft version and after testing I will create a separate PR.